java/conf/cobbler/snippets/minion_script

```diff
@@ -3,7 +3,13 @@
     <filename>uyuni-minion.sh</filename>
     <source>
         <![CDATA[
-cat <<EOF >"/etc/salt/minion.d/susemanager.conf"
+SALT_MINION_CONF_DIR="/etc/venv-salt-minion/minion.d"
+SALT_MINION_SERVICE="venv-salt-minion"
+if [ ! -d "$SALT_M_CONF_DIR" ]; then
```

Comment:
Shouldn't this be like this?

```suggestion
if [ ! -d "$SALT_MINION_CONF_DIR" ]; then
```

Reply:
ouch, I have no idea what moment the part was missing. Thanks for pointing